src/test/rustdoc/macros_2.rs

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Can you name this file `decl_macro.rs` instead?

Reply:
👍